py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -79,20 +91,101 @@ def for_token_id(cls) -> AtomicSwapCtrt.DBKey:
         @classmethod
         def for_token_balance(cls, addr: str) -> AtomicSwapCtrt.DBKey:
             """
-            for_token_balance [summary]
-
-            Args:
-                addr (str): [description]
+            for_token_balance returns the AtomicSwapCtrt.DBKey object for querying the token balance.
```

Comment:
The `Args` part of the docstring is missing here. The same for other DBKey classmethods.